### DISCUSSION TITLE
Doubts / Questions from user on trying shelf

### DISCUSSION THREAD

**Discussion by carlosvirreira** [Shelf-nu/shelf.nu#229] [Ideas]
I wish to tag **licenses** (For example, _Microsoft Office License_) for my office equipments; so the flow is Office License ABCDE is tied to LENOVO Office Equipment. 

> I understand that this feature is not available as of now, am I right to say that; or **am I missing out on how to tag one asset on another?**

> Showing the ID of the asset in the page doesn’t help users identify their product; plus it is not readable at all. I will prefer for it to show a short description of the asset; for a quick overview.

> Not sure on how this might work in shelf.nu, but if we could create master assets for example an LENOVO ABC master asset with the basic information, then I can select this LENOVO ABC and create my new laptop asset over this master asset; so I do not need to keep entering similar details over and over again.

**Comment by carlosvirreira** [comment]
@jurrejansen do you see some UX opportunities here from the feedback provided? 

@DonKoko master assets sound like some sort of a template for a certain asset class. Interesting to read

**Comment by jurrejansen** [comment]
> I understand that this feature is not available as of now, am I right to say that; or am I missing out on how to tag one asset on another?

Tag one asset to another, think we'll need a more detailed description of what they mean here. If you think about it assets can already be linked by giving them the same category or tag.

> Showing the ID of the asset in the page doesn’t help users identify their product; plus it is not readable at all. I will prefer for it to show a short description of the asset; for a quick overview.

In updated designs we already moved ID and we're using that space for indicating if an asset is published / archived and public / private. The short description is located under asset image.

![Screenshot 2023-06-16 at 10 43 24](https://github.com/Shelf-nu/shelf.nu/assets/76112272/717082ea-6b0e-4d0b-a3c6-85fa8356d2fe)

> Not sure on how this might work in shelf.nu, but if we could create master assets for example an LENOVO ABC master asset with the basic information, then I can select this LENOVO ABC and create my new laptop asset over this master asset; so I do not need to keep entering similar details over and over again.

Yeah this one i see being used in most other apps. You create the (master) asset and they you can have multiple asset instances of that same asset. It makes sense if you'd have 20 Magic Mouses or whatever. But it may also go against the simpleness / straightforwardness we're trying achieve with shelf.

Maybe duplicating assets would be an appropriate feature. It'll duplicate the asset with same category, tags, description, image, etc but with a new ID and QR. And / or maybe we'll have to consider master assets.

**Comment by DonKoko** [comment]
I am turning this into a discussion.

@jurrejansen can we translate this issue into something more actionable. If we have decided to go with the duplicate approach, lets make a feature request issue for Duplicate with description on how it works.
